### PR TITLE
Fix Renesas Compiler old

### PR DESCRIPTION
**Pull request by alanNz**
Fixed Renesas RX Family compiler to work with latest meson

### CODE REVIEW

**Comment by jpakkane** [comment]
Should this go in 0.53.2?

File: docs/markdown/snippets/add_release_note_snippets_here [hunk 1]
```diff
@@ -1,3 +1,5 @@
 ## Added `cpp_std` option for the Visual Studio C++ compiler
 Allows the use of C++17 features and experimental not-yet-standardized
 features. Valid options are `c++11`, `c++14`, `c++17`, and `c++latest`.
+
+## Fixed Renesas RX Family compiler to work with latest meson, added working cross file, fixed assembly file bug
```
Comment (jpakkane) [inline docs/markdown/snippets/add_release_note_snippets_here]: Please remove this. Release notes should only list new functionality, not bug fixes. The old text there is wrong and should not be there. I removed in trunk so you need to rebase.

**Comment by jpakkane** [comment]
Also, all pull requests should be targeted to master. The release branches are created automatically by cherrypicking based on the milestone tag.

**Comment by alanNz** [comment]
Oh ok I'll close this MR and send a new one into master

**Pull request closed without merging**